content/learn/publication-opportunities.md

```diff
@@ -0,0 +1,126 @@
+---
+Title: Pre- & Post-Publication Opportunities
+weight: 6400
+type: essay
+abstract:  "Funding, conferences, awards, and more for your Quire project"
+---
+
+## PRE-PUBLICATION
+
+#### Funding
```

Comment:
Did you go with four hashmarks because three looked too large? Let's give it a try anyways as it makes more sense in terms of page structure.